phpspec.yml

```diff
@@ -2,18 +2,15 @@
 # friends-of-phpspec/phpspec-code-coverage
 formatter.name: pretty
 suites:
-    default_suite:
-        namespace: FriendsOfPhpSpec\PhpSpec\CodeCoverage
-        psr4_prefix: FriendsOfPhpSpec\PhpSpec\CodeCoverage
+  default_suite:
+    namespace: FriendsOfPhpSpec\PhpSpec\CodeCoverage
+    psr4_prefix: FriendsOfPhpSpec\PhpSpec\CodeCoverage
 
 extensions:
   FriendsOfPhpSpec\PhpSpec\CodeCoverage\CodeCoverageExtension:
     format:
       - html
       - clover
-      - php
-      - text
     output:
-      html: build/coverage
-      clover: build/coverage.xml
-      php: build/coverage.php
+      html: coverage
```

Comment:
I had to change this because of the `deps=low` test and this bug: https://github.com/sebastianbergmann/php-code-coverage/issues/584

Reply:
Maybe we can add the `phpunit/php-code-coverage` v6.0.0 in the conflict section of the composer.json ? This way we will disallow to install the failing version.

Also, why removing the PHP format ? I'm not sure why there is all these formats generated ^^

Reply:
I also thought about adding the package in the conflict section. But I think using a `coverage` folder instead of a `build` folder is a better naming (because we don't really build anything when running our tests).

I removed the PHP format because I couldn't find a place were we are using it.

Reply:
Ok you are right using the conflict section just because of a `dist` folder seems a bit useless :smile:.